lego/apps/events/tests/test_events_api.py

```diff
@@ -8,6 +8,7 @@
 from rest_framework import status
 
 import stripe
+from dateutil.relativedelta import relativedelta
```

Comment:
Why not just use `django.utils.timezone.timedelta`?

Reply:
`timedelta` does not support months, weeks max. Probably why you ended up doing it the hacky way.